sites/next.skeleton.dev/src/content/docs/resources/contribute/documentation.mdx

```diff
@@ -138,9 +138,9 @@ export const tableProps: [
 ```
 
 ```html
-<Table label="Props" data="{tableData}" tonal>
+<table label="Props" data="{tableData}" tonal>
```

Comment:
This is incorrect - it should remain as `<Table>` (capital T) as this is a component, not an HTML element.